### PR TITLE
Stream Enrich: force jackson-databind to 2.9.3 for all projects

### LINKED ISSUE #3767: Stream Enrich: force jackson-databind to 2.9.3 for all projects
Related to #3744, we see a `java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.node.ArrayNode.<init>(Lcom/fasterxml/jackson/databind/node/JsonNodeFactory;Ljava/util/List;)` error at runtime when using R105 (which is equivalent to master at the time of writing).

Steps to reproduce:
```bash
git clone https://github.com/snowplow/snowplow
cd 3-enrich/stream-enrich
sbt "project kafka" assembly
# Run scala-stream-collector, stream-enrich 
```
Try to send test event like http://collector-server/i?e=pv. Stream-enrich will be down with this error as well as the previous bug with Kinesis.

### PR DESCRIPTION
**Pull request by romansalin**
Fix #3767 issue.

### CODE REVIEW

File: 3-enrich/stream-enrich/build.sbt [hunk 1]
```diff
@@ -17,6 +17,8 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.config,
   Dependencies.Libraries.slf4j,
   Dependencies.Libraries.log4jOverSlf4j,
+  Dependencies.Libraries.jacksonCbor,
```
[the remark this answers is not in the document]
Comment (romansalin) [reply 3-enrich/stream-enrich/build.sbt]: No, I have tested and it worked file without jacksonCbor as a common dependency. I'm going to revert it.

**Comment by romansalin** [comment]
@BenFradet, thanks, I signed the CLA.

**Comment by snowplowcla** [comment]
Confirmed! @yeonhoo has signed the Individual Contributor License Agreement. Thanks so much

**Comment by BenFradet** [comment]
integrated into #3799 thanks a lot!

**Pull request closed without merging**